48 feature add nf schema (#49)
ghga-de/nf-snvcalling: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

 ## v2.0.3 - dev

- add nf-schema and fix fasta.contain logic [#48](https://github.com/ghga-de/nf-snvcalling/pull/49)
- fix AD field error [#47](https://github.com/ghga-de/nf-snvcalling/pull/49)

## v2.0.2 - 12.11.2024

### `Fixed`

> - add cram/crai files to input list (assets/schema_input.json)
> - add only a warn about the indices does not exists in converttovcf.json (bin/convertToStdVCF.py)
> - change default arg for ENSEMBL VEP (conf/modules.config)
> - upgrade samtools version for cram handling (modules/local/grep_samplename.nf)
> - edit nextflow version to latest (nextflow.config)

Reviewer: ```suggestion
- add cram/crai files to input list (assets/schema_input.json)
- add only a warning about that indices do not exists in converttovcf.json (bin/convertToStdVCF.py)
- change default arg for ENSEMBL VEP (conf/modules.config)
- upgrade samtools version for cram handling (modules/local/grep_samplename.nf)
- edit nextflow version to latest (nextflow.config)
```
